runtime/lua/vim/lsp.lua

```diff
@@ -860,6 +860,8 @@ end
 ---                            Used on all running clients.
 ---                            The default implementation re-uses a client if name
 ---                            and root_dir matches.
+---             - bufnr (number)
```

Comment:
I imagine something needs to happen in order for the runtime files to be updated per what is part of the "docstring" here?

Reply:
>echo "Job failed, run ./scripts/gen_vimdoc.py and commit your doc changes"